sysconfig/jureca-booster/packages.yaml

```diff
@@ -26,13 +26,13 @@ packages:
         version: [0.19.8]
     hdf5:
         paths:
-            hdf5@1.8.20%intel: /usr/local/software/jurecabooster/Stages/2018a/software/HDF5/1.8.20-iimpi-2018a
+            hdf5@1.8.20%intel: /usr/local/software/jurecabooster/Stages/2018b/software/HDF5/1.8.20-iccifort-2019.0.117-GCC-7.3.0-serial
```

Comment:
I think it is misleading to set `variant` and leave the variant of the installed package unknown. If you hadn't set `buildable=False` it would install another version by default.